collections.md

```diff
@@ -344,10 +352,11 @@ You may even pass a callback as the default value. The result of the callback wi
 <a name="method-groupby"></a>
 #### `groupBy()` {#collection-method}
```

Comment:
Grrr. I'll talk to him about it. :)

Thank you for all your work on our collections, both in docs and implementation. :)